### PR TITLE
Fixed dependencies and changed name for Scrollable Calendar

### PR DESCRIPTION
**Pull request by boris-barath**
- made the component private,

- removed dependencies on bpk components from package-lock

- renamed comoponent to bpk-scrollable-calendar

- added "private": true to package.json

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -49,68 +49,69 @@ addDecorator(story => (
...
+  require('../packages/bpk-component-slider/stories');
+  require('../packages/bpk-component-spinner/stories');
+  require('../packages/bpk-component-star-rating/stories');
+  require('../packages/bpk-component-table/stories');
+  require('../packages/bpk-component-text/stories');
+  require('../packages/bpk-component-textarea/stories');
+  require('../packages/bpk-component-ticket/stories');
+  require('../packages/bpk-component-tile/stories');
+  require('../packages/bpk-component-tooltip/stories');
+  require('../packages/bpk-mixins/stories');
```
Comment (shaundon) [inline .storybook/config.js]: Do you know why this whole file has changed? It should just be the scrollable calendar line really.

File: packages/bpk-component-icon/src/withDescription.js [hunk 2]
```diff
@@ -25,14 +25,19 @@ import STYLES from './withDescription.scss';
...
     return (
       <span>
         <Component {...rest}>{children}</Component>
-        <span className={getClassName('bpk-icon-description')}>{description}</span>
+        <span className={getClassName('bpk-icon-description')}>
+          {description}
+        </span>
       </span>
     );
   };
```
Comment (shaundon) [inline packages/bpk-component-icon/src/withDescription.js]: This file isn't related to this PR so shouldn't have changed - could you revert the changes here?

File: package-lock.json [hunk 3]
```diff
@@ -5630,6 +5630,7 @@
         "require-from-string": "2.0.1",
         "rfc6902": "2.2.2",
         "supports-hyperlinks": "1.0.1",
+        "vm2": "github:patriksimek/vm2#468bc1e54e75e766b842830ea775669992a979e0",
```
Comment (shaundon) [inline package-lock.json]: This file isn't related to this PR so shouldn't have changed - could you revert the changes here?

**Comment by shaundon** [comment]
In terms of code it all looks good, the only issue is that CI is failing because some dependencies are outdated:

```
Checking Backpack cross dependencies...

Some Backpack cross dependencies are outdated  😱

bpk-component-scrollable-calendar depends on bpk-component-calendar ^4.2.38, it should be ^4.2.40
bpk-component-scrollable-calendar depends on bpk-mixins ^17.11.0, it should be ^17.11.2
bpk-component-scrollable-calendar depends on bpk-react-utils ^2.6.1, it should be ^2.6.3
```

If you put those versions in your `package.json`, you should be good.

**Comment by georgegillams** [comment]
Before we can merge this it will also need to be squashed. If you need a hand with that give us a shout 👍
